source/onboard/migrating-to-mattermost.rst

```diff
@@ -66,38 +66,6 @@ Sometimes systems running in parallel turn into a hard switch migration when a b
 
 When IT adopts management of Mattermost, often they will purchase the commercial version for additional compliance, access control, and scale features, in addition to high quality commercial support for upgrades and migrations. Teams can `purchase Mattermost Enterpise Edition with a credit card online <https://mattermost.com/pricing-self-managed/>`__ or `contact sales <https://mattermost.com/contact-us/>`__ to engage in an enterprise procurement process.
```

Comment:
Probably good to remove this paragraph as well

Reply:
here is the commit removing that one:

https://github.com/mattermost/docs/pull/6840/commits/e448daf2ac39f47d52963074511f95aa325e5b74

:pray:  thanks